commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -167,7 +169,15 @@ def message(self, answers: dict) -> str:
         if footer:
             footer = f"\n\n{footer}"
 
-        message = f"{prefix}{scope}: {subject}{body}{footer}"
+        message = f"{prefix}{scope}: {subject}"
+        message_len = len(message)
+        MESSAGE_LEN_LIMIT = 72
```

Comment:
Could we make it configurable?

Reply:
Also, in the comment https://github.com/commitizen-tools/commitizen/issues/191#issuecomment-1216952884, he proposes a reasonable solution. Although global variables might sometimes not be desired. I'm good with it in this case. @kevin1kevin1k @Woile  What do you think?